server.js

```diff
@@ -177,120 +175,120 @@ const server = http.createServer((req, res) => {
 server.listen(PORT, HOSTNAME, () => {
   console.log(`Server running at http://${HOSTNAME}:${PORT}/`);
 });
- */
+
 
 
 //TASK 06-09 CRUD For Slambook: 
-// const express = require('express');
-// const mongoose = require('mongoose');
-// const bodyParser = require('body-parser');
-// const cors = require('cors');
-
-// const PORT = 80;
-// const HOSTNAME = '0.0.0.0';
-
-// const app = express();
-
-// // Middleware
-// app.use(bodyParser.urlencoded({ extended: true }));
-// app.use(bodyParser.json());
-// app.use(cors());
-
```

Comment:
The message in the root route response should be updated to be more descriptive and meaningful.